pkg/streaming/commands/processor.go

```diff
@@ -29,7 +29,12 @@ func NewProcessorCommand(ctx context.Context, c *cli.Config) *cobra.Command {
 		Use:   "processor",
 		Short: "(experimental) processors apply functions to messages on streams",
 		Long: strings.TrimSpace(`
-<todo>
+Processors coordinate reading from input Streams and writing to output Streams
+with a Function or Container.
```

Comment:
This line is too short, the content from the next line should fill the remaining space until 80 chars is hit.

Unless this is intended to be a paragraph break, in which case a new line is missing.